### ISSUE TITLE
Doesn't work with Custom Docker locations

### ISSUE THREAD

**Issue by sr-tennisgazelle** [toptal/jenkins-job-trigger-action#15]
This assumes the docker daemon on a server to always be running on `unix:///var/run/docker.sock`

While this might be fair, this doesn't account for the use case of a custom github runner, where the env var DOCKER_HOST is set.
In cases where it is set (say, i'm running this on `ubuntu-latest` rather than on `custom-server-1`), then it will get past the docker build and creation of the container, but it will fail when it gets to the `docker run ...` cmd as it will have taken `DOCKER_HOST` as a parameter there.  

Basically, it's doomed if you provide the ENV VAR and it's doomed if you do not.
